content/authors/njxqlus.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Mikael Ahabalyants 
+slug: njxqlus
+avatar: njxqlus.jpg
+github_url: https://github.com/njxqlus
+---
```

Comment:
Please add a bio below this author information, as in our example

Reply:
Updated